packages/pluggableWidgets/barcode-scanner-web/src/ui/BarcodeScanner.scss

```diff
@@ -112,3 +112,12 @@
         }
     }
 }
+
+// Overwrite `atlas_core/web/core/_legacy/_mxui.scss` for this particular widget because otherwise
+// iOS Safari will in certain cases put the top and/or bottom bar on top of the overlay of this widget.
+.mx-dataview-content,
+.mx-scrollcontainer-wrapper:not(.mx-scrollcontainer-nested),
+.mx-tabcontainer-content,
+.mx-grid-content {
```

Comment:
Does this problem only occurs if we are using tab container?

Reply:
Maybe this property doesn't need to be "reset" for all the elements targeted here. Where do the elements mentioned, the top and bottom bar, come from?

Reply:
After debugging, it turns out that only `.mx-scrollcontainer-wrapper:not(.mx-scrollcontainer-nested)` is necessary for this change. In particular it's the middle scroll container.

> If we look at the documentation for this css property, it only minorly affects the experience of users and I don't think this is considered a breaking change. It does touch some legacy CSS that is used across mendix application, but based on what this change does and local testing, it does not seem to regress anything else.

Also please keep this in mind that the functional changes for users should be relatively minor: https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-overflow-scrolling